packages/server-wallet/src/protocols/open-channel.ts

```diff
@@ -46,304 +32,69 @@ export class ChannelOpener {
   public async crank(objective: DBOpenChannelObjective, response: WalletResponse): Promise<void> {
     const channelToLock = objective.data.targetChannelId;
 
-    let attemptAnotherProtocolStep = true;
+    this.store.transaction(async tx => {
+      const channel = await this.store.getLockedChannel(channelToLock, tx);
```

Comment:
Do you think `getAndLockChannel` might be a better name? As is, this line seems confusing because I have a channel to lock, and I then get the channel that is locked, but I seem to be missing the step that actually applies the lock. Checking the implementation, it seems like a lock _is_ applied.

Reply:
The lock is obtained in the store. So I think `getLockedChannel` makes sense, because the thing that comes back from the store is a locked channel?